src/AltinnCore/Common/Languages/ini/nb.ini

```diff
@@ -382,3 +382,13 @@ service_code = Tjenestekode
 edition_code = Tjenesteutgavekode
 subscription_text_helper = For å endre feilmeldingsteksten presentert for sluttbruker kan du benytte tekstnøkkeler som starter med "subscription_hook_error". Denne er tilgjengelig på språksiden i Altinn Studio.
 
+[receipt_container]
```

Comment:
I feel like "receipt" is a better section name

Reply:
Sure. I fix.